contentcuration/contentcuration/tests/test_exportchannel.py

```diff
@@ -558,6 +602,46 @@ def test_publish_no_modify_legacy_exercise_extra_fields(self):
             {"mastery_model": exercises.M_OF_N, "randomize": True, "m": 1, "n": 2},
         )
 
+    def test_qti_exercise_generates_qti_archive(self):
+        """Test that exercises with free response questions generate QTI archive files."""
+        qti_exercise = cc.ContentNode.objects.get(title="QTI Free Response Exercise")
+
+        # Check that a QTI archive file was created
+        qti_files = qti_exercise.files.filter(preset_id=format_presets.QTI_ZIP)
+        self.assertEqual(
+            qti_files.count(),
+            1,
+            "QTI exercise should have exactly one QTI archive file",
+        )
+
+        qti_file = qti_files.first()
+        self.assertIsNotNone(
+            qti_file.file_on_disk, "QTI file should have file_on_disk content"
+        )
+        self.assertTrue(
+            qti_file.original_filename.endswith(".zip"),
+            "QTI file should be a zip archive",
+        )
+
+    def test_qti_archive_contains_manifest_and_assessment_ids(self):
+
+        published_qti_exercise = kolibri_models.ContentNode.objects.get(
+            title="QTI Free Response Exercise"
+        )
+        assessment_ids = (
```

Comment:
confirm that the published assessment_item_ids in the channel database have been properly coerced to our QTI compatible ids.